packages/desktop-client/src/components/FinancesApp.js

```diff
@@ -253,96 +297,82 @@ class FinancesApp extends Component {
-  }
-
-  componentWillUnmount() {
-    this.cleanup();
-    window.removeEventListener('resize', this.handleWindowResize);
-  }
-
-  render() {
-    return (
-      <Router history={this.history}>
-        <CompatRouter>
-          <View style={{ height: '100%', backgroundColor: colors.n10 }}>
-            <GlobalKeys />
-
-            <View style={{ flexDirection: 'row', flex: 1 }}>
-              {!this.state.isMobile && <FloatableSidebar />}
-
+  return (
+    <Router history={patchedHistory}>
+      <CompatRouter>
+        <View style={{ height: '100%', backgroundColor: colors.n10 }}>
+          <GlobalKeys />
+
+          <View style={{ flexDirection: 'row', flex: 1 }}>
+            <FloatableSidebar />
+
+            <div
+              style={{
+                flex: 1,
+                display: 'flex',
+                flexDirection: 'column',
+                overflow: 'hidden',
+                position: 'relative',
+                width: '100%',
+              }}
+            >
+              <Titlebar
+                style={{
+                  WebkitAppRegion: 'drag',
+                  position: 'absolute',
+                  top: 0,
+                  left: 0,
+                  right: 0,
+                  zIndex: 1000,
+                }}
+              />
               <div
                 style={{
                   flex: 1,
                   display: 'flex',
-                  flexDirection: 'column',
-                  overflow: 'hidden',
+                  overflow: 'auto',
                   position: 'relative',
-                  width: '100%',
                 }}
               >
-                {!this.state.isMobile && (
-                  <Titlebar
-                    style={{
-                      WebkitAppRegion: 'drag',
-                      position: 'absolute',
-                      top: 0,
-                      left: 0,
-                      right: 0,
-                      zIndex: 1000,
-                    }}
-                  />
-                )}
-                <div
-                  style={{
-                    flex: 1,
-                    display: 'flex',
-                    overflow: 'auto',
-                    position: 'relative',
-                  }}
-                >
-                  <Notifications />
-                  <BankSyncStatus />
-                  <StackedRoutes isMobile={this.state.isMobile} />
-                  <Modals history={this.history} />
-                </div>
-                {this.state.isMobile && (
-                  <Switch>
-                    <Route path="/budget" component={MobileNavTabs} />
-                    <Route path="/accounts" component={MobileNavTabs} />
-                    <Route path="/settings" component={MobileNavTabs} />
-                  </Switch>
-                )}
+                <Notifications />
+                <BankSyncStatus />
+                <StackedRoutes />
+                {/*window.Actual.IS_DEV && <Debugger />*/}
```

Comment:
Why add this back?

Reply:
Looks like a merge conflict fail. Gone. 🪓